easypy/resilience.py

```diff
@@ -177,11 +178,16 @@ def retrying(times, acceptable=Exception, sleep=1, max_sleep=False, log_level=lo
     def wrapper(func):
         @wraps(func)
         def impl(*args, **kwargs):
-            return retry(
-                times, func, args, kwargs,
-                sleep=sleep, max_sleep=max_sleep,
-                acceptable=acceptable, log_level=log_level,
-                pred=pred)
+            custom_args = dict(
+                sleep=sleep,
+                max_sleep=max_sleep,
+                acceptable=acceptable,
+                log_level=log_level,
+                pred=pred
+            )
+            if unacceptable is not AUTO:
+                custom_args.update(unacceptable=unacceptable)  # We want to use 'retry' default value if a value was not specified
```

Comment:
Let's be consistent here. Either have `unacceptable=AUTO` in both `retry` and `retrying` or have `unacceptable=()` in both of them.

Reply:
I disagree, the reason I chose `AUTO` is to prevent a case in which we have to make sure the default value is the same in both `retry` and `retrying`. Since one is using the other, I think setting two default can only cause problems in the future (in case we decide to change the default value or its format).

Reply:
Then use `AUTO` in both.

Reply:
For what purpose? The reason i use `AUTO` in `retrying` is to make `retry`'s default value transparent to it.

Reply:
@DoronCohen - WDYT?

Reply:
`retry` and `retrying` should mirror each other's parameters. To the use they are two ways of achieving the same behavior - one by decorating a function (so retrying is a permanent part of its behavior), and the other is for ad-hoc situations, where the use-case requires a some function to be retried. Regardless, their parameter signature should be consistent, and if one uses `AUTO` to specify which exceptions are not allowed, then so should the other.
The fact that one function _uses_ the other should not affect the parameters.